components/UserListings/index.js

```diff
@@ -0,0 +1,27 @@
+import useSWR from "swr";
+import Flex from "@/components/Layout/Flex";
+import ListingCard from "../ListingCard";
+
+export default function UserList({ listings, starships }) {
+  function starshipURL(id) {
+    return starships.find((starship) => starship._id === id);
+  }
+
+  return (
+    <>
+      <Flex gap="1.5rem" flexWrap="wrap" direction="column" width="90%">
+        {listings.map((listing) => {
+          return (
+            <ListingCard
+              {...listing}
+              key={listing._id}
+              name={starshipURL(listing.starship).name}
+              preis={listing.preis}
```

Comment:
Please remember that writing your code in English will help other devs be able to read it! The "preis" is not a horrible example of German invading the code, as most people could probably figure out what it means, but, you know, be careful! :)

Reply:
That was an error on the backend (ops! xD). We are going to change some things in the backend this week and we will solve this problem as well!